src/keystats/keystatsActions.ts

```diff
@@ -0,0 +1,18 @@
+import { ThunkAction } from 'redux-thunk'
+import { API, GlobalState } from '../types'
+import { KeyStats } from './types'
+
+export const STATS_RECEIVED_ACTION = 'STATS_RECEIVED_ACTION'
+
+const statsReceivedAction = (keystats: KeyStats) => ({ 
+  type: STATS_RECEIVED_ACTION as typeof STATS_RECEIVED_ACTION, 
+  keystats 
+})
+
+export type StatsReceivedAction = ReturnType<typeof statsReceivedAction>
```

Comment:
Since this StatsReceivedAction is declared as a type and is exported, I would suggest that it should be moved to the types file?